expyfun/io/_parse.py

```diff
@@ -4,6 +4,39 @@
 
 import numpy as np
 import csv
+import ast
+import json
+from expyfun.stimuli import TrackerUD, TrackerBinom, TrackerDealer
```

Comment:
You might have a circular import problem. Nest the import to `Tracker*`, and to be clean make it a relative import, and you should be good.